### PR TITLE
add the copy of html cods

### PR DESCRIPTION
**Pull request by ahzamir**
I have made a survey form webpage, which contains.

- Input for the name.
- Input for the email.
- Input for the mobile number.
- making labels for them.
- choosing the role.
- leaving a comment.

Kindly review it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<html>
+    <h1 id=title>Heloo World</h1>
```
Comment (ahzamir) [inline index.html:2]: Please make the spelling correct for the "Hello World"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <p id="description">this is the survey form priject</p>
```
Comment (ahzamir) [inline index.html:3]: - [Optional] Add some descriptions about your page in this paragraph.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    <input type="checkbox" value="Front-End" name="choice">
+    <input type="checkbox" value="Back-End" name="choice">
+    <input type="checkbox" value="Database-Administration" name="choice">
+
+    <textarea name="" id="" cols="30" rows="10">
+
+    </textarea>
+
+    <button type="submit" id="submit">Submit</button>
+    </form>
```
Comment (ahzamir) [inline index.html:28]: - All your form sections(radio, checkbox, name, email) are in one line and will make your page looks informal and irregular. You can use "div" to separate every part of your form.

**Review by ahzamir — COMMENTED** [review]
### Changes Required ♻️
Hi [@ahzamir]
Nice job, you are almost there.



### (Highlights) Good Points: 👍

- Your project runs correctly. ✔️
- Your project work properly. ✔️
- Your project have the correct GitHub Flow✔️

### (Changes Required) Aspects to improve: ♻️

-  In the README file provide a simple description about your project.
- Your project code is not clean and understandable for viewers, Please add the linters configurations to your project, A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs.



-  Please check the inline comments below.

### [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**Review by vic778 — COMMENTED** [review]
# STATUS: INVALID!!!  :x: :x: :x: 

Hi @ahzamir ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code, please make sure your implement the following changes before you resubmit for review. 

## Required Changes ♻️
_Check the comments below._

- [ ] Please you have submitted a project that doesn't have a description. kindly add the description of your project saying what you have achieved in this project in order to allow the CR to understand your work. 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Mre55 — APPROVED** [review]
**Hi @ahzamir,**

# Status: APPROVED  🎊 💪

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**